### PR TITLE
HipChat improvements

### PR DESCRIPTION
**Pull request by jingx23**
- Added API V1 support (necessary for company tokens)
- Removed HipChat dependency => now using direct api calls

### CODE REVIEW

**Comment by KrauseFx** [comment]
Hey @jingx23, thanks for sending that pull request. 
It has a merge conflict, could you fix this?

File: README.md [hunk 1]
```diff
@@ -242,6 +242,9 @@ Send a message to **#channel** (by default) or a direct message to **@username**
 Send a message to **room** (by default) or a direct message to **@username** with success (green) or failure (red) status.
 
 ```ruby
+  ENV["HIPCHAT_API_TOKEN"] = "your api token"
```
Comment (KrauseFx) [inline README.md]: Please change to `"Your API token"`

**Comment by jingx23** [comment]
I see, git merge was unable to remove the hipchat dependency :/. Should be fine now :)

**Comment by KrauseFx** [comment]
Thanks @jingx23, looks great. It will be in the next release :+1:

**Comment by esttorhe** [comment]
:clap: :bow:

**Comment by KrauseFx** [comment]
@esttorhe are you using HipChat? Have you tried it? :smiley:

**Pull request merged**